application/controller/memo.go

```diff
@@ -0,0 +1,129 @@
+package controller
+
+import (
+	"errors"
+	"github.com/Nexters/myply/domain/memos"
+	"github.com/gofiber/fiber/v2"
+)
+
+type MemoController interface {
+	GetMemo(ctx *fiber.Ctx) error
+	AddMemo(ctx *fiber.Ctx) error
+}
+
+type memoController struct {
+	service *memos.Service
+}
+
+func NewMemoController(s *memos.Service) *MemoController {
+	var c MemoController
+	c = &memoController{service: s}
+	return &c
+}
+
+func (c *memoController) GetMemo(ctx *fiber.Ctx) error {
+	var resp Response
+
+	id := ctx.Params("id")
+
+	m, err := (*c.service).GetMemo(id)
+	if err != nil {
+		switch {
+		case errors.Is(err, memos.NotFoundException):
+			resp = Response{code: fiber.StatusNotFound, message: err.Error(), data: MemoResponse{}}
+			return ctx.Status(fiber.StatusNotFound).JSON(resp.toMap())
+		default:
+			resp = Response{code: fiber.StatusInternalServerError, message: err.Error(), data: MemoResponse{}}
+			return ctx.Status(fiber.StatusInternalServerError).JSON(resp.toMap())
+		}
+	}
+
+	// TODO: respond real data
+	memoResp := MemoResponse{memoId: m.Id, thumbnailURL: "", title: "", body: m.Body, keywords: []string{}}
+	resp = Response{
+		code:    fiber.StatusOK,
+		message: "",
+		data:    memoResp,
+	}
+
+	return ctx.Status(fiber.StatusOK).JSON(resp.toMap())
+}
+
+func (c *memoController) AddMemo(ctx *fiber.Ctx) error {
+	var resp Response
+
+	req := new(addRequest)
+	if err := ctx.BodyParser(req); err != nil {
+		return err
+	}
+
+	token := ctx.GetReqHeaders()["Device-Token"]
+	if token == "" {
+		resp = Response{
+			code:    fiber.StatusBadRequest,
```

Comment:
I copied google code [here](https://github.com/Nexters/myply/pull/42#discussion_r933956596).
I'll update it later!